libs/sdk-model/src/theme/index.ts

```diff
@@ -0,0 +1,140 @@
+// (C) 2020 GoodData Corporation
+
+/**
+ * @beta
+ */
+export type ThemeFontUri = string;
+
+/**
+ * @beta
+ */
+export interface IThemeTypography {
+    font?: ThemeFontUri;
+    fontBold?: ThemeFontUri;
+}
+
+/**
+ * Contains color string in hex format, e.g. #14b2e2
+ *
+ * @beta
+ */
+export type ThemeColor = string;
+
+/**
+ * @beta
+ */
+export interface IThemeColorFamily {
+    base: ThemeColor;
+    light?: ThemeColor;
+    dark?: ThemeColor;
+    contrast?: ThemeColor;
+}
+
+/**
+ * @beta
+ */
+export interface IThemePalette {
+    primary?: IThemeColorFamily;
+    error?: IThemeColorFamily;
+    warning?: IThemeColorFamily;
+    success?: IThemeColorFamily;
+}
+
+/**
+ * @beta
+ */
+export interface IThemeWidgetTitle {
+    color?: ThemeColor;
+    textAlign?: string;
+}
+
+/**
+ * @beta
```

Comment:
I know you's going to hate me but we need TSDoc for all properties of this object :/

we want to eventually generate API docs from code; users should know what the properties do and what are the accepted values.